activity-metrics/open-issues.md

```diff
@@ -1,10 +1,30 @@
-# Open Issues
-
 ## 1. Description
-Number of open issues
+
+These are the issues considered 'open' at some given point in time. The definition of 'open' differs from platform to platform and should be pre-defined.
+
+Note: do not confuse this with the opened tickets in a specific period of time regardless their current status. The open issues metric is a snapshot while the opened issues one needs of a period of time.
+
+### Parameters
+
+Mandatory: 

```

Comment:
I think we also need to parameterize the periodic aggregation : daily/weekly/monthly/quarterly/annually? 
And also the range of reporting.  Date X to Date Y.

Reply:
So, for each periodic aggregation, you capture this metric.

Reply:
But this is not issues opened during a certain period, but issues that are in state open at a given point in time (as opposed to issues being in a closed state at that same point in time, for example). 

If you want, I could clarify in the description that the metric is not about issues that people open during a certain period, but issues that are still open at a certain point in time. What do  you think?